app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/MainViewModel.kt

```diff
@@ -32,54 +32,54 @@ class MainViewModel : ViewModel() {
 
     fun globalData() {
         GlobalScope.launch(context = Dispatchers.Main) {
-            var gdp : GlobalDataProvider = GlobalDataProvider()
+            val gdp : GlobalDataProvider = GlobalDataProvider()
 
             /**
              * Call API to retrieve specific global covid19 data
              */
-            var gCases: ArrayList<String>? = gdp.getGlobalCovidData("total_cases")
-            var gActive: ArrayList<String>? = gdp.getGlobalCovidData("active_cases")
-            var gTotalDeath: ArrayList<String>? = gdp.getGlobalCovidData("total_deaths")
-            var gRecovered: ArrayList<String>? = gdp.getGlobalCovidData("total_recovered")
-            var gNewCases: ArrayList<String>? = gdp.getGlobalCovidData("new_cases")
-            var gNewDeaths: ArrayList<String>? = gdp.getGlobalCovidData("new_deaths")
-            var gSeriousCritical: ArrayList<String>? = gdp.getGlobalCovidData("serious_critical")
-            var gCasesPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("total_cases_per_1m_population")
-            var gDeathPerMillion: ArrayList<String>? = gdp.getGlobalCovidData("deaths_per_1m_population")
-            var gStatistics: ArrayList<String>? = gdp.getGlobalCovidData("statistic_taken_at")
```

Comment:
Changed as the variables remained constant so the variable would be better off as a val